### PR TITLE
Adding clusterservicebrokers permissions to customer-admin-cluster.

### PR DESCRIPTION
**Pull request by kwoodson**
Adding `clusterservicebroker` to the `customer-admin-cluster` clusterrole.

With this additional binding was successfully able to deploy an azure postgresql db.
```
[cloud-user@master-000000 ~]$ oc get servicebinding azure-sql-12-0-6fg9w-q2v5v -o yaml
apiVersion: servicecatalog.k8s.io/v1beta1
kind: ServiceBinding
metadata:
  creationTimestamp: 2018-10-26T16:48:11Z
  finalizers:
  - kubernetes-incubator/service-catalog
  generateName: azure-sql-12-0-6fg9w-
  generation: 1
  name: azure-sql-12-0-6fg9w-q2v5v
...
    conditions:
    - lastTransitionTime: 2018-10-26T16:52:27Z
      message: The instance was provisioned successfully
      reason: ProvisionedSuccessfully
      status: "True"
      type: Ready
```

```
[cloud-user@master-000000 ~]$ oc get serviceinstance -o yaml
apiVersion: v1
items:
- apiVersion: servicecatalog.k8s.io/v1beta1
  kind: ServiceInstance
  metadata:
    creationTimestamp: 2018-10-26T16:48:11Z
    finalizers:
    - kubernetes-incubator/service-catalog
    generateName: azure-sql-12-0-
    generation: 1
    name: azure-sql-12-0-6fg9w
...
  status:
    asyncOpInProgress: false
    conditions:
    - lastTransitionTime: 2018-10-26T16:52:27Z
      message: The instance was provisioned successfully
      reason: ProvisionedSuccessfully
      status: "True"
      type: Ready
...
```

### CODE REVIEW

**Comment by kwoodson** [comment]
@kargakis, is this a flake on e2e test?
```
time="2018-10-26T17:01:51Z" level=info msg="applying arm template deployment" resourceGroup=ci-op-fg04cjjb-8474e
time="2018-10-26T17:01:53Z" level=info msg="waiting for arm template deployment to complete" resourceGroup=ci-op-fg04cjjb-8474e
time="2018-10-26T17:16:53Z" level=debug msg="400 Bad Request: Failed to apply requestFuture#WaitForCompletion: context has been cancelled: StatusCode=200 -- Original Error: context deadline exceeded" resourceGroup=ci-op-fg04cjjb-8474e
time="2018-10-26T17:16:53Z" level=fatal msg="containerservice.OpenShiftManagedClustersClient#CreateOrUpdate: Failure sending request: StatusCode=0 -- Original Error: autorest/azure: error response cannot be parsed: \"400 Bad Request: Failed to apply request\\n\" error: json: cannot unmarshal number into Go value of type azure.RequestError" resourceGroup=ci-op-fg04cjjb-8474e
exit status 1
2018/10/26 17:16:54 Container setup in pod e2e-azure failed, exit code 1, reason Error
2018/10/26 17:17:04 Container test in pod e2e-azure failed, exit code 1, reason Error
```

File: pkg/addons/data/ClusterRole.rbac.authorization.k8s.io/customer-admin-cluster.yaml [hunk 1]
```diff
@@ -2,4 +2,17 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: customer-admin-cluster
-rules: []
+rules:
+- apiGroups:
+  - servicecatalog.k8s.io
+  attributeRestrictions: null
```
Comment (enj) [inline pkg/addons/data/ClusterRole.rbac.authorization.k8s.io/customer-admin-cluster.yaml]: remove

**Pull request merged**